src/main/java/eu/dissco/core/datacitepublisher/domain/datacite/DcPublisher.java

```diff
@@ -4,8 +4,16 @@
 
 @Value
 public class DcPublisher {
-  String name = "Distributed System of Scientific Collections";
-  String publisherIdentifier = "https://ror.org/0566bfb96";
-  String publisherIdentifierScheme = UriScheme.ROR.getSchemeName();
-  String schemeUri = UriScheme.ROR.getUri();
+  String name;
+  String publisherIdentifier;
+  String publisherIdentifierScheme;
+  String schemeUri;
+
+  public DcPublisher(){
+    name = "Distributed System of Scientific Collections";
```

Comment:
can't be static or it wont seralize to json :upside_down_face: 

The reason it's like this is because now we serialize datacite responses, so we'd get a jsonprocessingexception if these values ever differ from what we get from datacite (e.g. if the publisher name changes)

I'll add the fields as properties